### ISSUE TITLE
ListView has been removed from React Native

### ISSUE THREAD

**Issue by prabhjodhOYO** [APSL/react-native-keyboard-aware-scroll-view#371]
The ListView component has been removed from the React Native and hence the project is giving an error during build. Attached below is the screenshot of the error.

![Simulator Screen Shot - iPhone X - 2019-07-08 at 16 45 06](https://user-images.githubusercontent.com/46446002/60806239-1e7f2880-a1a0-11e9-9f05-7defdda03766.png)

**Comment by KiranUppunda** [comment]
Hi, @prabhjodhOYO .

Did you find any break-through ?

**Comment by prabhjodhOYO** [comment]
@KiranUppunda The issue has been fixed in version v0.8.0. Please update the dependencies to resolve the issue.

**Comment by kevinjacyna** [comment]
@prabhjodhOYO what needs to be updated to v0.8.0?

**Comment by DheereshSinghKarki** [comment]
@prabhjodhOYO  please tell me which dependencies to i need to update and the issue is fixed in version v0.8.0 of what i could not get it?

**Comment by DheereshSinghKarki** [comment]
> > @prabhjodhOYO please tell me which dependencies to i need to update and the issue is fixed in version v0.8.0 of what i could not get it?
> 
> @DheereshSinghKarki Upgrade react-native-keyboard-aware-scroll-view to latest version

but i have not installed this package and neither did i used listview in any of my files

**Comment by DheereshSinghKarki** [comment]
@prabhjodhOYO Sir i have not used this package or listview in my code

![Screenshot (2)](https://user-images.githubusercontent.com/25513447/65953239-2a0d8300-e461-11e9-95f7-f423c7973f6a.png)

**Comment by wedusa** [comment]
I have already replaced this library, but every time I see a warning after npm install. Help only removing this in `react-native-implementation.js`:
```
Object.defineProperty(module.exports, 'ListView', {
    configurable: true,
    get() {
      invariant(
        false,
        'ListView has been removed from React Native. ' +
          'See https://fb.me/nolistview for more information or use ' +
          '`deprecated-react-native-listview`.',
      );
    },
  });
```

**Comment by kunal-invezta** [comment]
@prabhjodhOYO your solution solved my issues

**Comment by avinashpatnaik** [comment]
> > > @prabhjodhOYO please tell me which dependencies to i need to update and the issue is fixed in version v0.8.0 of what i could not get it?
> > 
> > 
> > @DheereshSinghKarki Upgrade react-native-keyboard-aware-scroll-view to latest version
> 
> but i have not installed this package and neither did i used listview in any of my files

Remove all the references to native-base. Do a quick search for the same in node_modules. That seems to be the work-around.
